docs/snp.md

```diff
@@ -105,6 +111,10 @@ All script created guests can be stopped by running the following command:
 ./snp.sh stop-guests
 ```
 
+User created SNP guest via guest-name option can be stopped with the `--guest-name` option as follows:
+```
+./snp.sh stop-guests --guest-name <user-guest-name>
+```
```

Comment:
I don't see this as part of the code, are you meaning to add it?

Reply:
In the documentation, I was just highlightling a way to stop guest for a specific guest.

`--guest-name`  option sets the IMAGE env. variable for that specific guest, which stops the running IMAGE env variable as per the stop_guests() function.

Let me know if we don't want to show this in documentation?

Reply:
My bad, I misunderstood how that worked, you can ignore that.

Reply:
No worries, @DGonzalezVillal , thanks for approving this PR.
Hi @ryansavino , please let me know if any changes are required in this PR.